src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -25,6 +26,9 @@ interface CommunityAPIService {
     val primeProviderName = referral.intervention.dynamicFrameworkContract.primeProvider.name
     return "$description for $contractTypeName Referral ${referral.referenceNumber} with Prime Provider $primeProviderName\n$url"
   }
+
+  fun camelCase(word: String) =
+    word.lowercase().replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }
```

Comment:
this appears to just capitalize the first letter of the input string? what's it supposed to be doing?

Reply:
will make explicit as per your later comment

val status = when(event.type) {
  APPROVED -> "Approved"
  SUBMITTED -> "Submitted"
}